backup.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+#
+# Script made for backup PostgreSQL database from local (${POSTGRES_HOST}=127.0.0.1)
+# or remote host. Created backup puts on S3 storage. On completion script calls
+# notification script hooks/00-webhook.sh which sends report to given Telegram Chat
+#
+# TODO (siameseoriental) Implement error handling
+
+set -euo pipefail
+IFS=$'\n\t'
+
+# Will be name of directory in backet dd-mm-yyyy_hh-mm-ss
+timestamp="$(date date +%FT%T%Z)"
+
+# Export stuff
+export PGPASSWORD=${POSTGRES_PASSWORD}
+
+# Will create base backup
+echo "Creating backup of ${POSTGRES_DB} database..."
```

Comment:
Add more info here (host, port, user, opts). It would be nice in case of debugging.

Reply:
Done.